### ISSUE TITLE
compute_1D_tuning_curves creates numeric dataframe column names 

### ISSUE THREAD

**Issue by jolson1129** [pynapple-org/pynapple#200]
This is bad practice and creates issues when attempting to call the columns (key errors). I suggest converting to strings.

**Comment by gviejo** [comment]
Hi Can you expand on the issue?

**Comment by jolson1129** [comment]
Essentially the same issue as here: https://stackoverflow.com/questions/27156278/index-pandas-dataframe-by-column-numbers-when-column-names-are-integers

Key errors when attempting to access columns

**Comment by gviejo** [comment]
In pynapple >= 0.4, you can label columns of a TsdFrame with strings and number and retrieve the right column or group of columns with the `.loc` method. Like `tsdframe.loc['a']` or `tsdframe.loc[[0,2]]`. You can also do it without the `.loc` for strings like `tsdframe['a']`.
The syntax `tsdframe[0]` will return the first row of the TsdFrame object and it's a behavior similar to numpy. In fact, it's similar to `.iloc` in pandas.

**Comment by jolson1129** [comment]
I'm sorry Guillaume, I typo'd in the title. Now fixed.

I'm referring to the output of 
process.tuning_curves.compute_1d_tuning_curves

The TsdFrame column names of the compute_1d_tuning_curves output directly imports the TsGroup keys.
Your code examples (and maybe it used to be forced?) used integer keys for TsGroup - which then makes the output of this to be columns with numeric names, which, as documented in that link I shared, lead to some unusual issues when attempting to use .loc

All that being said, I'm pretty new, and if you think this behavior is best, by all means. I just stumbled on accessing because of it.
